### PR TITLE
Feature 2 - complete cart component

### LINKED ISSUE #2: Create cart component
Sub-task of #1

Implementation: React component should display item image, price, quantity, etc....

### PR DESCRIPTION
**Pull request by ssw2114**
### Assignee Tasks

* [ ] added unit tests (or none needed)
* [ ] written relevant docs (or none needed)
* [ ] referenced any relevant issues (or none exist)

### Guidelines

Please add a description of this Pull Request's motivation, scope, outstanding issues or potential alternatives, reasoning behind the current solution, and any other relevant information for posterity.

---

Closes #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+TESTING PUSH TO MASTER
```
Comment (ssw2114) [inline README.md:1]: Why is this here?

Comment (alena-ioffe) [reply README.md:1]: Looks good

**Review by alena-ioffe — APPROVED** [review]
